RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# Grakn Client Python
+
+{ Please add the release notes here }
+
+{ Please add instructions on how to install } 
```

Comment:
Can you also add the instructions to install in this template? It should change very rarely, right?

Reply:
@haikalpribadi I'm not sure I know how to best phrase it. The only thing that comes to mind:
"To install Grakn Client Python, run `pip install -U grakn-client`"

Reply:
Just explain how you think best. We'll review it then. Just make sure you provide complete instructions and point them to the docs where necessary.

ps: what is `-U` for?

Reply:
@haikalpribadi `-U` is `upgrade` which makes sure user always installs the latest Grakn (in case he already has one installed)

Reply:
is that really the command we should tell users to do?

Reply:
@haikalpribadi yep. is there something wrong with it? 🤔 
We can also include [this link](https://dev.grakn.ai/docs/client-api/python#installation) to docs

Reply:
@haikalpribadi addressed in https://github.com/graknlabs/client-python/pull/36/commits/7b4db5b595d6de3b39df4bb97297881cbeab0ce6